ansible/templates/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       - "127.0.0.1:8536:8536"
     restart: always
     environment:
-      - RUST_LOG=error
+      - RUST_LOG="warn,lemmy_server=info,lemmy_api=info,lemmy_api_common=info,lemmy_api_crud=info,lemmy_apub=info,lemmy_apub_receive=info,lemmy_db_queries=info,lemmy_db_schema=info,lemmy_db_views=info,lemmy_db_views_actor=info,lemmy_db_views_moderator=info,lemmy_routes=info,lemmy_utils=info,lemmy_websocket=info"
```

Comment:
This is very verbose, but essentially it means "show warnings for dependencies, and info logs for all lemmy_* crates". Unfortunately there doesnt seem to be any shorter way to express this.

Reply:
Ya seems unavoidable.